### PR TITLE
chore: update docker to build and expose frontend

### PR DESCRIPTION
**Pull request by jakubkoci**
I tried to build and start the Next.js frontend together with backend in docker and expose it on other port but it fails to show styles and images. That must be related to the Next.js build step inside the Docker, we perhaps don't copy all needed files into the container 🤷‍♂️ 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,25 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn workspace @ssi-trust-registry/backend build",
-    "validate": "yarn workspace @ssi-trust-registry/backend validate && yarn workspace @ssi-trust-registry/frontend validate",
+    "fe:build": "yarn workspace @ssi-trust-registry/frontend build",
+    "be:build": "yarn workspace @ssi-trust-registry/backend build",
+    "build": "yarn be:build && yarn fe:build",
```
Comment (niall-shaw) [inline package.json:10]: Is there a reason that we don't use `concurrently` for builds? Only validate and start?

**Review by niall-shaw — APPROVED** [review]

File: packages/frontend/package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "validate": "yarn format:check && yarn lint"
   },
   "dependencies": {
+    "daisyui": "^4.0.4",
```
Comment (niall-shaw) [inline packages/frontend/package.json:15]: Why is `daisyui` no longer in dev deps?

File: package.json (hunk 1 above)
Comment (jakubkoci) [reply package.json:10]: No particular reason, we could do it too

File: packages/frontend/package.json (hunk 2 above)
Comment (jakubkoci) [reply packages/frontend/package.json]: Ah, good point. First I thought it's an issue with daisy. Moved back to dev deps.

File: package.json (hunk 1 above)
Comment (jakubkoci) [reply package.json:10]: It seems that the build in docker got stuck with concurrently, so I keep it without that for now.

**Review by niall-shaw — APPROVED** [review]

**Pull request merged**